packages/envied_generator/pubspec.yaml

```diff
@@ -11,9 +11,8 @@ dependencies:
   envied: ^1.0.0
   build: ^2.4.1
   code_builder: ^4.8.0
-  dart_style: ^2.3.2
-  source_gen: ^1.4.0
-  analyzer: ">=5.1.0 <7.0.0"
+  source_gen: ^2.0.0
+  analyzer: ^7.0.0
```

Comment:
This is too harsh. `source_gen` [2.0.0](https://pub.dev/packages/source_gen/changelog) only requires
```yaml
analyzer: ">=6.9.0 <8.0.0"
```
so please lower the constraint.